index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```

Comment:
Can we somehow simplify this html file, maybe having something similar like:
https://github.com/kbss-cvut/fta-fmea-ui/blob/e1a243c59f12c55fae8fe66ad8b558bd104b0263/index.html